cpp/include/ray/api.h

```diff
@@ -39,6 +39,14 @@ class Ray {
   template <typename T>
   static ObjectRef<T> Put(const T &obj);
 
+  /// Get a single object from the object store.
+  /// This method will be blocked until all the objects are ready.
+  ///
+  /// \param[in] object The object reference which should be got.
```

Comment:
```suggestion
  /// \param[in] object. The object reference which should be returned.
```

Reply:
I found no the "." separator in other code, just like core_worker.h?